hatter/README.md

```diff
@@ -0,0 +1,43 @@
+![Barsoom](http://barsoom.se/barsoom.png)
+
+# Hatter
+
+The developers take turns being the "hatter" (or "hat").
+
+They are the designated person(s) keeping an eye on incoming exceptions and requests from the rest of the organisation. This lets the rest of the dev team focus on planned tasks.
+
+## 🎩 The Hatter's Guide 🎩
+
+### Your first priority is to triage
+
+The hatter triages incoming requests (from Slack to the "Hatten" Trello) and the dev dashboard (mainly logged exceptions), routing them where they belong.
+
+They should look at requests pretty soon after they arrive, to ensure we don't miss anything important, and so we can ask for clarifying questions while the requester has context.
+
+They should look at exceptions pretty soon after they arrive as well. When possible, they should make a ticket, link to it, and resolve the exception, keeping the dashboard green. This makes it easier to spot new exceptions and to determine priority of ticketed ones.
```

Comment:
It could, but often there is no time to do that, and other tasks are more prioritised. This way we have a clear indication that:

* The issue has been triaged
* The dashboard is green
* New issues are clearly visible and not drowning in noise

Reply:
I initially wrote something like "just fix it or…" but removed it to keep the text shorter – thought it was implicit. Could make it explicit :)